app/developers/templates/developers/deliver/_dependency_banner.html

```diff
@@ -0,0 +1,21 @@
+{% from "govuk_frontend_jinja/components/notification-banner/macro.html" import govukNotificationBanner %}
+{# todo: its not nice that we have to pass around so much contexts - we shouldnt need this much to route #}
+{% macro dependency_banner(error, message, grant_id, collection_id, section_id, form_id) %}
```

Comment:
Nice to pull this out into a macro 👍 

Do we maybe want this in some kind of macros/partials dir? We have a 🤏  precedent for those elsewhere.

In my head macros=...macros, partials=includes. But we haven't actually applied that principle so we're already a bit scattered, oops. One for us to clean up separately maybe.

Reply:
https://github.com/communitiesuk/funding-service/pull/427/commits/60be2d1b7865d35bcae4091c90cbc6ae49c72ec6